### PR TITLE
Fixed rows' indices in batch DataGrid column

### PR DESCRIPTION
**Pull request by chives**

### CODE REVIEW

**Review by szymach — APPROVED** [review]

File: src/Component/DataSource/Driver/Doctrine/ORM/ORMDriver.php [hunk 1]
```diff
@@ -110,9 +110,10 @@ public function getResult(array $fields, ?int $first, ?int $max): Result
         $result = new ORMPaginator($query);
```
Comment (szymach) [inline src/Component/DataSource/Driver/Doctrine/ORM/ORMDriver.php:110]: Shouldn't this be simply called `$paginator` and then `$indexedResult` could be `$result`?

**Comment by chives** [comment]
@fsi-open

**Review by szymach — APPROVED** [review]

**Comment by chives** [comment]
@fsi-open

**Comment by chives** [comment]
@fsi-open

**Pull request merged**